atmega32u4_firmware/src/main.ino

```diff
@@ -0,0 +1,12 @@
+const int FT4232_RESET_PIN = 12;
+
+void setup(){
+    pinMode(12, OUTPUT);
+    digitalWrite(12, LOW);
+    delay(5000);
+    digitalWrite(12, HIGH);
+}
```

Comment:
Unused const? then `pinMode` and `digitalWrite` use that same value?

Reply:
fixed